modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceReplacementProducts.testcase

```diff
@@ -184,6 +184,109 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-12579. As a buyer, I can view the discontinued SKU shown in the product's detail replacement tab"
+	@priority = 3
+	test CanReplacementSkusShownInProductDetails {
+		property portal.acceptance = "false";
+
+		task ("Given 5 products are set as Discontinued with a same replacement product ABS Sensor") {
```

Comment:
Is it possible to use this JSON macro here? 

CommerceJSONProductsAPI._patchCommerceSku();